Documentation/creating-new-package.md

```diff
@@ -0,0 +1,29 @@
+# Creating new .NET IoT package
+
+This is a list of steps which need to happen in order to create new package version of .NET IoT.
```

Comment:
Same here, this is not techinically true, the steps bellow are only for releasing a package to Nuget